### PR TITLE
Developer mode option

### PR DESCRIPTION
**Pull request by philn**

### CODE REVIEW

**Comment by clopez** [comment]
> In 2.28 CCache should be automatically detected without explicit CMake option.

Did you tested this with Yocto?

**Comment by philn** [comment]
Yes:
```
-- Enabling ccache: Setting ccache prefix for compiler.
```

and in rules.ninja:
```
command = /edited/path/to/wpewebkit/2.26.4-r0/recipe-sysroot-native/usr/bin/ccache /edited/path/to/tmp/work/aarch64-poky-linux/wpewebkit/2.26.4-r0/recipe-sysroot-native/usr/bin/aarch64-poky-linux/aarch64-poky-linux-g++  $DEFINES $INCLUDES $FLAGS -MD -MT $out -MF $DEP_FILE -o $out -c $in
....
```

**Comment by philn** [comment]
This is with ToT though, I think the patches are not in 2.28 yet, I have added them in the wiki.

**Comment by clopez** [comment]
Ok. Thanks

File: recipes-browser/wpewebkit/wpewebkit.inc [hunk 1]
```diff
@@ -24,6 +24,7 @@ PACKAGECONFIG ??= "fetchapi indexeddb mediasource video webaudio webcrypto woff2
 # WPE features
 PACKAGECONFIG[accessibility] = "-DENABLE_ACCESSIBILITY=ON,-DENABLE_ACCESSIBILITY=OFF,atk at-spi2-atk"
 PACKAGECONFIG[bubblewrap] = "-DENABLE_BUBBLEWRAP_SANDBOX=ON,-DENABLE_BUBBLEWRAP_SANDBOX=OFF,bubblewrap xdg-dbus-proxy bubblewrap-native xdg-dbus-proxy-native libseccomp"
+PACKAGECONFIG[developer-mode] = "-DDEVELOPER_MODE=ON,-DDEVELOPER_MODE=OFF,wayland-native wayland-protocols"
```
Comment (philn) [inline recipes-browser/wpewebkit/wpewebkit.inc]: This might need wpebackend-fdo too, I'm trying to understand why. Please don't merge yet.

Comment (philn) [reply recipes-browser/wpewebkit/wpewebkit.inc]: OK it's needed for the "tooling" backends.

**Comment by JoshuaWatt** [comment]
I didn't see this before pushing my pull request, but OE-core correctly handles ccache, so we shouldn't be specifying manually in recipes (it breaks if a user doesn't want to use ccache, e.g. when using icecream). See #148

**Comment by clopez** [comment]
This is merged right? Closing.
(Re-open if not)

**Comment by philn** [comment]
It needs a rebase.

**Pull request merged**